### ISSUE TITLE
using containerName something different than "media"

### ISSUE THREAD

**Issue by perosb** [JimBobSquarePants/UmbracoFileSystemProviders.Azure#5]
Hi there,

In our setup, the filesystemproviders configuration is using a containerName = "my-media".
The installation is also configured to use the CloudImageService in ImageProcessor.

While this seem to work for images, an uploaded video, .mp4, throws a 404 error.
The video is stored with the path /my-media/1001/thevideo.mp4

The video seem to work if I request it directly using /media/1001/thevideo.mp4

I suspect this has to do with the configuration method which always uses the default "media" as a path.

```
FileSystemVirtualPathProvider.ConfigureMedia();
```

which in our case, possible should have read the containerName instead?
```
FileSystemVirtualPathProvider.ConfigureMedia("my-media");
```

**Comment by JimBobSquarePants** [comment]
Oh, I thought I just had `media` as a default parameter. I'll double check the code.

**Comment by perosb** [comment]
For now I have just added an additional call to
```
FileSystemVirtualPathProvider.ConfigureMedia("my-media");
```
Everything seem to work, though it is still regestering your default.
I'll fork it and try with just the one.

**Comment by JimBobSquarePants** [comment]
Cheers! :+1:  It should just be the case of passing the container as a parameter.

**Comment by perosb** [comment]
Correct. But there are no other downsides of registering yet another VirtualPath?

**Comment by JimBobSquarePants** [comment]
Hold on... 

Rereading this it seems to me that the behaviour is correct. The container name is simply the name of the blob container that you have set up on Azurenot the path that the media should be requested under. 

Everything you save to Umbraco should be stored and retrieved under the path `/media`

**Comment by perosb** [comment]
But the path stored in umbraco database is the ```containerName``` which is also what the ```Umbraco.Media``` will resolve the assets to.

So all these settings need to match up:
```
<service prefix="my-media" name="CloudImageService" type="ImageP...ocessor.Web">
<add key="containerName" value="my-media" />
FileSystemVirtualPathProvider.ConfigureMedia("my-media");
```

**Comment by JimBobSquarePants** [comment]
Ah right... Then we just need to pass the parameter then. Should be a simple update.

**Comment by Jeavon** [comment]
I think I've fixed this up now, please check out the v0.5.1 prerelease and let me know
